### PR TITLE
Exposed nuke missile sprites to lint testing

### PR DESCRIPTION
**Pull request by Mailaender**
A followup of https://github.com/OpenRA/OpenRA/pull/10723.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs [hunk 1]
```diff
@@ -22,10 +22,15 @@ class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientation
 	{
 		[WeaponReference]
 		[Desc("Weapon to use for the impact.",
-			"But also image to use for the missile.",
-			"Requires an 'up' and a 'down' sequence on the image.")]
+			"Also image to use for the missile.")]
 		public readonly string MissileWeapon = "";
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs]: Based on the description and the blank default I assume this wants a `[FieldLoader.Require]` attribute.

Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs]: Added.

**Comment by RoosterDragon** [comment]
:+1: Looks fine to me.

File: OpenRA.Mods.Common/Effects/NukeLaunch.cs [hunk 2]
```diff
@@ -36,12 +37,13 @@ public class NukeLaunch : IEffect
 		WPos pos;
 		int ticks;
 
-		public NukeLaunch(Player firedBy, string name, WeaponInfo weapon, string weaponPalette,
+		public NukeLaunch(Player firedBy, string name, WeaponInfo weapon, string weaponPalette, string up, string down,
```
Comment (pchote) [inline OpenRA.Mods.Common/Effects/NukeLaunch.cs]: `upSequence` and `downSequence` please, to make it clear what these actually are.

**Comment by pchote** [comment]
This doesn't seem to work.  The linter doesn't complain when I set `MissileUp: foo` on the Temple of Nod, but it crashes ingame.

**Comment by Mailaender** [comment]
Fixed.

**Comment by obrakmann** [comment]
:+1:

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/24b8e3391eefc96caa94fa35e08346bf9b13c585)

**Pull request merged**